### ISSUE TITLE
Building OpenStack with Linux Bridge is failed in the commit `2a6de8d`.

### ISSUE THREAD

**Issue by TsutomuNakamura** [tsuna-server/build-server-ansible#48]
Building OpenStack with Linux Bridge is failed in the commit `2a6de8d`.
I will run both Ansible in a commit `2a6de8d` and `6763f93(current master branch)` to clarify which role in each commit is wrong.

Example sinario 1.
```
 Commit 2a6de8d    | Commit 3475ab9
 roles)            | roles)
  *placement       |   placement
   nova            |  *nova
   neutron         |  *neutron
   horizon         |  *horizon
...
```

If this instruction was succeeded, we can assume nova, neutron or horizon... in commit `2a6de8d` have wrong instructions.

**Comment by TsutomuNakamura** [comment]
test01
```
* 2a6de8d
to placement
* 3475ab9
from nova
```

**Comment by TsutomuNakamura** [comment]
test01
```
* 2a6de8d
to nova
* 6763f93
from neutron
```

Result OK.

**Comment by TsutomuNakamura** [comment]
Role nova in commit `2a6de8d` has something wrong instructions.
We can fetch both configurations and verify the differences between them.

**Comment by TsutomuNakamura** [comment]
There are some missing configurations in `/etc/nova/nova.conf` in the commit `2a6de8d`.
This issue will be solved by modifying like below.

* /etc/nova/nova.conf (in controller node)
```diff
 [DEFAULT]
 log_dir = /var/log/nova
 lock_path = /var/lock/nova
 state_path = /var/lib/nova

 transport_url = rabbit://openstack:${PASSWORD}@${CONTROLLER_NODE}:5672/
 my_ip = 172.29.0.1
+vif_plugging_is_fatal = false
+vif_plugging_timeout = 0
```

* /etc/nova/nova.conf (in compute node)
```diff
 [DEFAULT]
 log_dir = /var/log/nova
 lock_path = /var/lock/nova
 state_path = /var/lib/nova

 transport_url = rabbit://openstack:${PASSWORD}@${CONTROLLER_NODE}:5672/
+vif_plugging_is_fatal = false
+vif_plugging_timeout = 0
 ...
 [api_database]
+connection = sqlite:////var/lib/nova/nova_api.sqlite
 ...
 [database]
+connection = sqlite:////var/lib/nova/nova.sqlite
 ...
 [neutron]
 auth_url = http://${CONTROLLER_NODE}:5000
 auth_type = password
 project_domain_name = Default
 user_domain_name = Default
 region_name = RegionOne
 project_name = service
 username = neutron
 password = ${PASSWORD}
-service_metadata_proxy = true
-metadata_proxy_shared_secret = ${PASSWORD}
 ...
 [vnc]
 enabled = true
 server_proxyclient_address = $my_ip
 server_listen = 0.0.0.0
-novncproxy_base_url = http://${SOME_WRONG_HOSTNAME}:6080/vnc_auto.html
+novncproxy_base_url = http://${CONTROLLER_NODE}:6080/vnc_auto.html

```

**Comment by TsutomuNakamura** [comment]
This issue has been solved at `f15a39c`.
Close.
